addons/html_editor/static/src/editor/core/delete_plugin.js

```diff
@@ -83,6 +83,7 @@ export class DeletePlugin extends Plugin {
             targetNode: selection.anchorNode,
             targetOffset: selection.anchorOffset,
         });
+        this.shared.makeStep();
```

Comment:
Also, nitpicking but personally I'd call it `addStep` rather than `makeStep` since we're not just making it but adding it to the history.

Reply:
done